feature-utils/poly-look/src/react-components/tabs.css

```diff
@@ -20,7 +20,7 @@
 
 .tabs .tab-button.active {
   padding-bottom: 1px;
-  color: var(--color-text-dark) !important;
+  color: var(--theme-secondary-color) !important;
```

Comment:
I'm as usual confused with these changes in color; I guess they're adapted from the design and/or were incorrect before, since they were (apparently) different and they are the same now. Nonetheless, it seems unrelated to the main theme of the PR.

Reply:
Used tabs in the PR and the colors should be theme now